boards/adi/max32672fthr/doc/index.rst

```diff
@@ -0,0 +1,212 @@
+.. _max32672_fthr:
+
+MAX32672FTHR
+############
+
+Overview
+********
+The MAX32672FTHR is a rapid development platform
+that helps engineers quickly implement complex sensor
+solutions using the MAX32672 Arm® Cortex®-M4. The
+board also includes the MAX8819 PMIC for battery and
+power management. The form factor is a small, 0.9in by
+2.6in, dual row header footprint that is compatible with
+Adafruit® FeatherWing peripheral expansion boards. The
+board includes an OLED display, a RGB indicator LED,
+and a user pushbutton. The MAX32672FTHR provides
+a power-optimized flexible platform for quick proof-ofconcepts
+and early software development to enhance
+time to market.
+
+The Zephyr port is running on the MAX32672 MCU.
+
+.. image:: img/max32672fthr_img1.jpg
+   :align: center
+   :alt: MAX32672FTHR Front
+
+.. image:: img/max32672fthr_img2.jpg
+   :align: center
+   :alt: MAX32672FTHR Back
```

Comment:
ditto earlier comment: not convinced you need the back here either

Reply:
Thanks.